wp_insert_rows.php

```diff
@@ -17,18 +18,21 @@
 	*
 	*  }
 	*
-	*  wp_insert_rows($insert_arrays);
+	*  wp_insert_rows($insert_arrays, $wpdb->tablename, true, "column_name_of_Primary_key");
 	*
 	*
 	* @param array $row_arrays
 	* @param string $wp_table_name
+	* @param boolean $update
+	* @param string $prikeyname
 	* @return false|int
 	*
 	* @author	Ugur Mirza ZEYREK
+	* @contributor Travis Grenell
 	* @source http://stackoverflow.com/a/12374838/1194797
 	*/
 	 
-	function wp_insert_rows($row_arrays = array(), $wp_table_name) {
+function wp_insert_rows($row_arrays = array(), $wp_table_name, $update, $prikeyname) {
```

Comment:
You are correct, that is a better idea. You are certainly welcome, your
code helped me immensely!

On Wed, Jun 22, 2016 at 2:49 AM, uğur mirza zeyrek <notifications@github.com
> wrote:

> In wp_insert_rows.php
> <https://github.com/mirzazeyrek/wordpress-multiple-insert/pull/3#discussion_r68001829>
> :
>
> >  	* @source http://stackoverflow.com/a/12374838/1194797
> >  	*/
> >  	
> > -	function wp_insert_rows($row_arrays = array(), $wp_table_name) {
> > +function wp_insert_rows($row_arrays = array(), $wp_table_name, $update, $prikeyname) {
>
> An existing user should not have to modify old codes. Therefore, I think
> adding default values as "$update=false, $primary_key = null" should be the
> way it supposed to be.
>
> BTW, I'm very glad to see your contribution on this repo :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mirzazeyrek/wordpress-multiple-insert/pull/3/files/cf59c123acf80f4cf41468005d230bd0a2e0ca34#r68001829>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAvBXWjrGaP9D1IbzTIfT1mRpkRhFBhFks5qONr-gaJpZM4I7V6Z>
> .
>